src/query/plan/rewrite/index_lookup.hpp

```diff
@@ -444,7 +444,7 @@ class IndexLookupRewriter final : public HierarchicalLogicalOperatorVisitor {
 
   bool PreVisit(Foreach &op) override {
     prev_ops_.push_back(&op);
-    return false;
+    return true;
```

Comment:
This won't visit the internal brancch of the `FOREACH` operator, right?

Reply:
it will, my mistake. will correct to rewrite the branch with the new root.